_data/navbar.yml

```diff
@@ -9,3 +9,5 @@
   url: /documentation
 - key: titles.forum
   url: https://forum.antennapod.org/
+- key: titles.events
+  url: /events
```

Comment:
I think it would fit better before or after "contribute". Then the last item stays the "external" link. This also ensures that the "contribute"-type links are together, while the "user-support"-type links are in the end.